### PR TITLE
Update IP Pool CRD to contain disableBGPExport

### PR DESCRIPTION
**Pull request by coutinhop**
## Todos
- [x] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: go.mod
Comment (mgleung) [inline go.mod:42]: We should point this to our API once https://github.com/projectcalico/api/pull/40 is merged

**Review by mgleung — APPROVED** [review]
LGTM should just need an update once the dependencies are merged

**Review by caseydavenport — CHANGES_REQUESTED** [review]
We need a UT in the clientv3 suite to confirm that modifying this field is respected (e.g., when written, can be queried and the value is correct)

Probably goes in this suite: https://github.com/projectcalico/libcalico-go/blob/master/lib/clientv3/ippool_e2e_test.go

**Comment by coutinhop** [comment]
> We need a UT in the clientv3 suite to confirm that modifying this field is respected (e.g., when written, can be queried and the value is correct)
> 
> Probably goes in this suite: https://github.com/projectcalico/libcalico-go/blob/master/lib/clientv3/ippool_e2e_test.go

Got it, will do! Thanks @caseydavenport

File: go.mod
Comment (caseydavenport) [reply go.mod:42]: This still needs updating.

**Review by caseydavenport — APPROVED** [review]

**Pull request merged**
